src/field_specials.c

```diff
@@ -961,9 +961,9 @@ static bool8 IsPlayerInFrontOfPC(void)
     GetXYCoordsOneStepInFrontOfPlayer(&x, &y);
     tileInFront = MapGridGetMetatileIdAt(x, y);
 
-    return (tileInFront == METATILE_BrendansMaysHouse_BrendanPC_Off
-         || tileInFront == METATILE_BrendansMaysHouse_MayPC_Off
-         || tileInFront == METATILE_Building_PC_Off);
+    return ((tileInFront == METATILE_BrendansMaysHouse_BrendanPC_On || tileInFront == METATILE_BrendansMaysHouse_BrendanPC_Off)
+         || (tileInFront == METATILE_BrendansMaysHouse_MayPC_On || tileInFront == METATILE_BrendansMaysHouse_MayPC_Off)
+         || (tileInFront == METATILE_Building_PC_On || tileInFront == METATILE_Building_PC_Off));
```

Comment:
```suggestion
    return (tileInFront == METATILE_BrendansMaysHouse_BrendanPC_On
         || tileInFront == METATILE_BrendansMaysHouse_BrendanPC_Off
         || tileInFront == METATILE_BrendansMaysHouse_MayPC_On
         || tileInFront == METATILE_BrendansMaysHouse_MayPC_Off
         || tileInFront == METATILE_Building_PC_On
         || tileInFront == METATILE_Building_PC_Off);
```

Reply:
Looks uglier, but alright.

Reply:
I disagree, it looked less readable before.

Reply:
Tile for screen off followed by tile for screen on. Makes sense to me and it's not really unreadable, but to each their own 😆